Kernel/Random.cpp

```diff
@@ -63,6 +65,14 @@ KernelRng::KernelRng()
 
             this->resource().add_random_event(value, i % 32);
         }
+    } else {
```

Comment:
You'll need to check that the system actually has a HPET timer, probably by checking `TimeManagement::m_can_query_precise_time`

Reply:
Now it uses HPET if it's found, RTC otherwise.